sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -6,27 +6,38 @@
 
 from enum import Enum
 from typing import TYPE_CHECKING
+from ._generated.models import ContentProperties
 
 if TYPE_CHECKING:
-    from ._generated.models import ManifestChangeableAttributes, ManifestAttributesBase
+    from ._generated.models import ManifestAttributesBase
     from ._generated.models import RepositoryProperties as GeneratedRepositoryProperties
+    from ._generated.models import TagProperties as GeneratedTagProperties
 
 
 class ContentPermissions(object):
     def __init__(self, **kwargs):
```

Comment:
if part of public API, docstring with description + keywords? (but maybe you know this already or are working on it.)